platformLibs/src/platform/mingw/iconv.def

```diff
@@ -0,0 +1,24 @@
+depends = posix
+package = platform.iconv
+linkerOpts = -Wl,-Bstatic -liconv -Wl,-Bdynamic
+
+---
+#include <iconv.h>
```

Comment:
Better `#include "iconv.h"` or `headers = iconv.h`

Reply:
Even better:
```
headers = iconv.h
headerFilter = NOTHING
```
Now `libiconv_*` variants are not visible.